### PR TITLE
i18n(es): Update `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by K1ri86**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- What does this PR change? 
- Update #4883

### CODE REVIEW

File: src/content/docs/es/guides/view-transitions.mdx [hunk 1]
```diff
@@ -248,7 +248,7 @@ Las siguientes opciones te permiten controlar aún más cuándo ocurre la navega
 
 ### Previniendo la navegación del lado del cliente
 
-Existen casos en los que no se puede navegar a través del CSR ya que ambas páginas involucradas deben utilizar el enrutador `<ViewTransitions />` para evitar una recarga de página completa. También puede que no desees la navegación del lado del cliente en cada cambio de navegación y prefieras una navegación de página tradicional en rutas selectas en su lugar.
+Existen casos en los que no se puede navegar a través del client-side routing ya que ambas páginas involucradas deben utilizar el enrutador `<ViewTransitions />` para evitar una recarga de página completa. También puede que no desees la navegación del lado del cliente en cada cambio de navegación y prefieras una navegación de página tradicional en rutas selectas en su lugar.
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:251]:
```suggestion
Existen casos en los que no se puede navegar a través del enrutamiento del lado del cliente ya que ambas páginas involucradas deben utilizar el enrutador `<ViewTransitions />` para evitar una recarga de página completa. También puede que no desees la navegación del lado del cliente en cada cambio de navegación y prefieras una navegación de página tradicional en rutas selectas en su lugar.
```

**Review by Waxer59 — CHANGES_REQUESTED** [review]
Buena PR 🤠, te dejo una sugerencia

File: src/content/docs/es/guides/view-transitions.mdx (hunk 1 above)
Comment (K1ri86) [reply src/content/docs/es/guides/view-transitions.mdx]: Perdon, era demasiado tarde y al parecer no hice nada 😅

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 💜

**Pull request merged**